cat/ptx.cat

```diff
@@ -0,0 +1,54 @@
+PTX
```

Comment:
As you wrote in some comment, the old ptx version is basically the new one without the proxy stuff and one smalle change (a `| po` somewhere). So please apply all the changes I proposed to `ptx-proxy` also here

Reply:
Same update applied to ptx.cat.